### PR TITLE
Update docs - Happy new year 2018!

### PR DESCRIPTION
**Pull request by ntulsy**

### CODE REVIEW

**Comment by vjeux** [comment]
Thanks!

File: docs/why-prettier.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ What usually happens once people are using Prettier is that they realize that th
 
 * “I want to write code. Not spend cycles on formatting.”
 * “It removed 5% that sucks in our daily life - aka formatting”
-* “We're in 2017 and it's still painful to break a call into multiple lines when you happen to add an argument that makes it go over the 80 columns limit :(“
+* “We're in 2018 and it's still painful to break a call into multiple lines when you happen to add an argument that makes it go over the 80 columns limit :(“
```
Comment (azz) [inline docs/why-prettier.md]: This is a quote and shouldn't be changed.

Comment (vjeux) [reply docs/why-prettier.md]: Oops!

Comment (azz) [reply docs/why-prettier.md]: Changed it back in c00c1cb6e045b71ad3d4e0fb55af256e85042e13

**Pull request merged**